client/src/common/components/GeoLocation/utils.tsx

```diff
@@ -0,0 +1,29 @@
+/* global GeolocationPositionError */
+import React, { ReactNode } from 'react';
+
+// Types
+import { LocationInfo } from '../../../types';
+
+export const renderConditionalJSX = (
```

Comment:
function definitely should be renamed :) it's not just any conditional jsx, your function, first of all, returns location info (otherwise request status) and name of your function should show it

Reply:
Would renderDataOrReqStatus be better?

Reply:
`Data` means it  can be any data, but in your case it should be location data. `renderLocationName\renderLocationLabel` is enough

Reply:
But I plan to reuse it. So `Data` could be `LocationName` or `LocationCurrWeather`, or `LocationHourlyWeather`, or maybe even `LocationMap` (if I would manage to insert any) or something. So maybe `renderLocationData` then?